config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -516,19 +520,20 @@ periodics:
       - -c
       - |
         set -euo && \
-        git clone https://github.com/jetstack/cert-manager.git && \
-        git clone https://github.com/cert-manager/test-infra.git && \
-        cd test-infra/aws && \
+        ls && \
+        pwd && \
+        cd /home/prow/go/src/github.com/jetstack/cert-manager/test-infra/aws && \
```

Comment:
I _think_ the path might be `/home/prow/go/src/github.com/cert-manager/test-infra/aws`? (i.e `/home/prow/go/src/github.com/<org>/<repo>/aws`
Same on line 536 and 812, 823 in the other file.

Reply:
This was the initial path so I added that but yeah I'm pretty sure we'll need a few more PRs to figure out where stuff is :)